spec/features/admin/pages_spec.rb

```diff
@@ -1,7 +1,13 @@
 require 'rails_helper'
 
 describe 'admin pages' do
-  let!(:account){ FactoryGirl.create :confirmed_account }
+  let!(:account) do
+    FactoryGirl.create(:confirmed_account).tap do |account|
+      Theblog::Role.find_by(name: :editor).accounts << account
```

Comment:
I would introduce a factory to define account roles dynamically, like account_editor, account_moderator, etc